tools/integration_tests/operations/operations_test.go

```diff
@@ -134,11 +134,18 @@ func TestMain(m *testing.M) {
 	// Set up test directory.
 	setup.SetUpTestDirForTestBucketFlag()
 	// Set up flags to run tests on.
+	// Note: GRPC related tests will work only if you have allow-list bucket.
 	flags := [][]string{
 		// By default, creating emptyFile is disabled.
 		{"--implicit-dirs=true"},
 		{"--implicit-dirs=false"},
-		{"--experimental-enable-json-read=true", "--implicit-dirs=true"}}
+		{"--experimental-enable-json-read=true", "--implicit-dirs=true"},
```

Comment:
Nit: shall we add a check that bucket name has allowlist prefix and log the error if not?

Reply:
It's hard to check for allowlisted prefix. Any way test will fail while mounting only.

Reply:
Discussed offline: We will skip this as this won't add much value and would need unnecessary cleanups later. The mount failure error for --client_protocol=grpc flag will hint the same.